extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -527,14 +527,18 @@ export class ImageViewer {
   onZoom_(scale, deltaX, deltaY, animate) {
     const newScale = this.boundScale_(scale, true);
     if (newScale == this.scale_) {
-      return;
+      return Promise.resolve();
```

Comment:
does that mean we can guarantee its always a promise? theres a few calls in this codebase that just calls `then` right away assuming that these functions always return a Ptomise